spec.bs

```diff
@@ -2832,6 +2868,20 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
        navigable=], or the result of [=checking if unloading is user-canceled=] for |navigable|'s
        [=navigable/active document=]'s [=Document/inclusive descendant navigables=] otherwise.
 
+  Add a step after step 22.8.2 (Let |finalSandboxFlags| be the [=set/union=]...) that reads:
+
+  3. If |sourceSnapshotParams|' [=source snapshot params/target fenced frame config=] is not null,
+     and if |sourceSnapshotParams|' [=source snapshot params/target fenced frame config=]'s [=fenced
+     frame config/effective sandboxing flags=] is not null, then set |finalSandboxFlags| to the
+     [=set/union=] of |finalSandboxFlags| and |sourceSnapshotParams|' [=source snapshot
+     params/target fenced frame config=]'s [=fenced frame config/effective sandboxing flags=]'
+     [=effective sandboxing flags/value=].
+
+     Note: This ensures that the |finalSandboxFlags| are *at least* as restrictive as the [=fenced
+     frame config/effective sandboxing flags=] defined in the [=source snapshot params/target fenced
+     frame config=]. A separate check will make sure that |finalSandboxFlags| is not more
```

Comment:
Please describe exactly where (and ideally link to, if you can, just for readability and navigability) exactly the separate check is.

Reply:
Added a link to the `should navigation response to navigation request be blocked by sandboxing flags?` algorithm.